src/ResourceManager/Network/Commands.Network/packages.config

```diff
@@ -18,7 +18,7 @@
   <package id="Microsoft.Data.OData" version="5.6.4" targetFramework="net45" />
   <package id="Microsoft.Data.Services.Client" version="5.6.4" targetFramework="net45" />
   <package id="Microsoft.Net.Http" version="2.2.28" targetFramework="net45" />
-  <package id="Microsoft.Rest.ClientRuntime" version="2.3.5" targetFramework="net45" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.8" targetFramework="net452" />
```

Comment:
why net452 and not just net45?

would 2.3.6 use just net45? if thats the case, lets use 2.3.6 instead